src/Symfony/Bridge/Twig/Extension/FormExtension.php

```
@@ -95,6 +103,62 @@ public function getTests()
     }
 
     /**
+     * @internal
+     */
+    public function __get($name)
+    {
+        if ('renderer' === $name) {
```

we should throw an exception for any other name